src/devices/Mpu9250/Mpu9250.cs

```diff
@@ -0,0 +1,983 @@
+            }
+        }
+
+        /// <summary>
+        /// Get or set the accelerometer bandwidth
+        /// </summary>
+        public AccelerometerBandwidth AccelerometerBandwidth
+        {
+            get => _accelerometerBandwidth;
+
+            set
+            {
+                WriteRegister(Register.ACCEL_CONFIG_2, (byte)value);
+                _accelerometerBandwidth = (AccelerometerBandwidth)ReadByte(Register.ACCEL_CONFIG_2);
+                if (_accelerometerBandwidth != value)
+                    throw new IOException($"Can set {nameof(AccelerometerBandwidth)}, desired value {value}, stored value {_accelerometerBandwidth}");
+            }
+        }
+
+        /// <summary>
+        /// Get the real accelerometer bandwith. This allows to calculate the real
+        /// degree per second
+        /// </summary>
+        public float AcceleromterConversion
+        {
+            get
+            {
+                float val = 0;
+                switch (AccelerometerRange)
+                {
+                    case AccelerometerRange.Range02G:
+                        val = 2.0f;
+                        break;
+                    case AccelerometerRange.Range04G:
+                        val = 4.0f;
+                        break;
+                    case AccelerometerRange.Range08G:
+                        val = 8.0f;
+                        break;
+                    case AccelerometerRange.range16G:
+                        val = 16.0f;
+                        break;
+                    default:
+                        break;
+                }
+
+                val = (val * Gravity) / Adc;
+                return val / (1 + SampleRateDivider);
+            }
+        }
+
+        /// <summary>
+        /// Get the acccelerometer in G
+        ///    +Z   +Y
+        ///  \  |  /
+        ///   \ | /
+        ///    \|/
+        ///    /|\
+        ///   / | \
+        ///  /  |  \
+        ///         +X
+        /// </summary>
+        public Vector3 Accelerometer => GetRawAccelerometer() * AcceleromterConversion;
+
+        private Vector3 GetRawAccelerometer()
+        {
+            Span<byte> rawData = stackalloc byte[6] { 0, 0, 0, 0, 0, 0 };
+            Vector3 ace = new Vector3();
+            ReadByteArray(Register.ACCEL_XOUT_H, rawData);
+            ace.X = BinaryPrimitives.ReadInt16BigEndian(rawData);
+            ace.Y = BinaryPrimitives.ReadInt16BigEndian(rawData.Slice(2));
+            ace.Z = BinaryPrimitives.ReadInt16BigEndian(rawData.Slice(4));
+            return ace;
+        }
+
+        /// <summary>
+        /// Set or get the accelerometer low power mode
+        /// </summary>
+        public AccelerometerLowPowerFrequency AccelerometerLowPowerFrequency
+        {
+            get { return (AccelerometerLowPowerFrequency)ReadByte(Register.LP_ACCEL_ODR); }
+            set { WriteRegister(Register.LP_ACCEL_ODR, (byte)value); }
```

Comment:
We should probably do some validation here to make sure that the value is a valid AccelerometerLowPowerFrequency.

Reply:
As it's an enum, I'm not cheking. If it would be a byte or equivalent, I would